src/screens/admin/AddFields.tsx

```
@@ -0,0 +1,475 @@
+      const options = [...prev.options];
+      options.splice(idx, 1);
+      return { ...prev, options };
+    });
+  };
+
+  const validateForm = () => {
+    const newErrors: Record<string, string> = {};
+    if (!form.label.trim()) newErrors.label = 'Label is required';
+    if (!form.name.trim()) newErrors.name = 'Name is required';
+    if (!form.type) newErrors.type = 'Type is required';
+    if (form.isEditable === undefined || form.isEditable === null) newErrors.isEditable = 'Editable is required';
```

ESLint

[error] Replace `····if·(form.isEditable·===·undefined·||·form.isEditable·===·null)·` with `↹↹if·(form.isEditable·===·undefined·||·form.isEditable·===·null)⏎↹↹↹`

(prettier/prettier)